src/deroff.rs

```diff
@@ -530,6 +533,27 @@ impl Deroffer {
     }
 }
 
+fn deroff_files(files: &[String]) -> std::io::Result<()> {
+    for arg in files {
+        eprintln!("processing deroff file: {}", arg);
+
+        let mut file = File::open(arg)?;
+        let mut string = String::new();
+        if arg.ends_with(".gz") {
+            let mut decoder = Decoder::new(file).unwrap();
```

Comment:
What are the conditions for which `Decoder::new` would return an `Err`? (When would the `unwrap` panic?)

Reply:
unsure...the documentation for libflate seems to be lacking :( https://docs.rs/libflate/0.1.25/libflate/gzip/struct.Decoder.html

Reply:
Blerg. Yeah. Looks like it's IO errors or the file not being compressed via `DEFLATE(8)`. Probly okay to leave for now. Don't know how often this will happen.